### ISSUE TITLE
Make cli flag an actual flag

### ISSUE THREAD

**Issue by nyggus** [nyggus/makepackage#10]
In the current version of the `makepackage` package, to create a package with the command-line interface, you need to add `cli` argument to the `makepackage` command, like this:
```shell
$ makepackage newpackage cli
```
This does not look as a typical flag used in Python for command-line interface, does it? So, I propose to make it
```shell
$ makepackage newpackage --cli
```
so the flag is what we consider a command-line flag indeed. To keep backwards compatibility, the `makepackage` command should also accept the `cli` flag, and, because why not, the `-cli` flag. 

To summarize, three versions of the flag will be accepted:
* `cli`
* `-cli`
* `--cli`

Note that they cannot be confused with anything else, so introducing this change will not be dangerous in any way. I think we can keep the three flags; perhaps in the future we can remove the `cli` flag, though I really do not see the point, so I am reluctant to the idea.

@hp0404 and @AlixBernard , I'd be obliged for your thoughts on the issue.

**Comment by hp0404** [comment]
If it can't break anything and is convenient, why not? I think all three should be accepted

**Comment by nyggus** [comment]
Done.

**Comment by Dutcho** [comment]
I believe this issue is still open in the `main` branch, which `pip install` will get from `pypi`.

As-is, the usage is `makepackage <name> ["cli"]` **only**, `--cli` isn't recognized. There's an `improve-cli-flag` branch, which I assume fixes this, but it needs merging into `main` to install from `pypi`.

**Comment by nyggus** [comment]
Yes, indeed, thanks for spotting this! I will take care of this soon.

**Comment by nyggus** [comment]
After all, it's truly done and closed.
